TP-Sampler/src/view.js

```diff
@@ -194,7 +194,7 @@ View.prototype = {
 
   createSampleSlots: function(device) {
     var sSampleSize = sampleSize>=1? Math.floor(sampleSize): 0,
```

Comment:
Not sure what the convention in this repo is, but usually we have spaces around math and conditionals.
```suggestion
    var sSampleSize = sampleSize >= 1 ? Math.floor(sampleSize) : 0,
```